storage/migrations/02_runtimes.up.sql

```diff
@@ -207,6 +207,19 @@ CREATE TABLE chain.evm_token_balance_analysis
 );
 CREATE INDEX ix_evm_token_balance_analysis_stale ON chain.evm_token_balance_analysis (runtime, token_address, account_address) WHERE last_download_round IS NULL OR last_mutate_round > last_download_round;
 
+CREATE TABLE chain.evm_contracts
+(
+  runtime runtime NOT NULL,
+  contract_address oasis_addr NOT NULL,
+  creation_tx HEX64 NOT NULL,
+  creation_bytecode BYTEA NOT NULL,
```

Comment:
Also can be null because it will be potentially populated in a separate analyzer. Please add a comment here explaining this.
Please make sure the API code can handle NULLs here.